hearth/lib/hearth/cbor/tagged.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Hearth
+  module Cbor
+    Tagged = Struct.new(:tag, :value)
```

Comment:
Do these need docs? (Is this public?)

Reply:
I initially thought this should be considered private, but in theory the `CBOR.decode` can return it (if the data has a Tagged item that is not one of the types we support like Time) - I've removed this file and move the Tagged class and added documentation.